embeddings/use-cases/outlier-detection/bq-vector-search-log-outlier-detection.ipynb

```diff
@@ -0,0 +1,1420 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pxhwaND4_IvS"
+   },
+   "source": [
+    "## Generate text embeddings for logs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "gjy7jR0T_eAL"
+   },
+   "source": [
+    "### Create BigQuery Cloud resource connection"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hpQqsysu_zCr"
+   },
+   "source": [
+    "You need to create a Cloud resource connection to enable BigQuery Dataframes to interact with Vertex AI services"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "dbWYeyEzii71"
+   },
+   "outputs": [],
+   "source": [
+    "CONN_NAME = \"bqdf-llm-embeddings\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "5_ko_ngACnPd"
+   },
+   "outputs": [],
+   "source": [
+    "!bq mk --connection --location={LOCATION} --project_id={PROJECT_ID} \\\n",
+    "    --connection_type=CLOUD_RESOURCE {CONN_NAME}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "hS0kzr2gD0-h"
+   },
+   "source": [
+    "Retrieve the connection service account ID"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "IG3KaOQb_doE"
+   },
+   "outputs": [],
+   "source": [
+    "!bq show --connection {PROJECT_ID}.{LOCATION}.{CONN_NAME}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "D143kdK7Hjpf"
+   },
+   "source": [
+    "Copy the service account ID in the following parameter:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "ryD1ufggF6Fj"
+   },
+   "outputs": [],
+   "source": [
+    "# Copy the value of `serviceAccountId` field in last cell's output\n",
+    "CONN_SA_ID = \"[bq-connection-service-account]\"  # @param {type:\"string\"}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "pa3lCS2hHqv-"
+   },
+   "source": [
+    "Give that service account access to Vertex AI"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "h-SoTNgKHpvg"
+   },
+   "outputs": [],
+   "source": [
+    "!gcloud projects add-iam-policy-binding {PROJECT_ID} --member='serviceAccount:{CONN_SA_ID}' --role='roles/aiplatform.user' --condition=None --no-user-output-enabled"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "8HJgiz91IBHz"
+   },
+   "source": [
+    "### Define the LLM model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create a remote model in BigQuery using the above BigQuery connection. The remote endpoint is set as `textembeddings-gecko` LLM model in Vertex AI."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "LLM_ENDPOINT = \"textembedding-gecko\"\n",
```

Comment:
Let's add the version here: textembedding-gecko@001 or textembedding-gecko@003 to make sure we use the same version of embedding models for both base table and query.